tool/gravity/cli/update.go

```diff
@@ -132,6 +125,13 @@ func updateTrigger(
 
 	if !manual {
 		req.leaderParams = []string{constants.RpcAgentUpgradeFunction}
+		// attempt to schedule the master agent on this node but do not
+		// treat the failure to do so as critical
+		req.leader, err = findLocalServer(*cluster)
+		if err != nil {
+			log.Errorf("Failed to determine local node: %v.",
```

Comment:
`log.Warnf` if it's not critical

Reply:
Will change in the next PR so I don't have to wait for CI again.